examples/inspection/plot_partial_dependence.py

```diff
@@ -213,3 +225,61 @@
 plt.subplots_adjust(top=0.9)
 
 plt.show()
+
+##############################################################################
+# Individual Conditional Expectation (ICE) Plots
+# ----------------------------------------------
+#
+# One of the main limitations of ICE plots is that the plot gets overcrowded
+# if many ICE curves are drawn. Due to this, we will use a random sample of
+# 50 instances for the ICE plot.
+#
+# ICE plots are achieved by setting ``individual=True``.
+# Let's now compute the ICE plots for this Gradient Boosting model:
+
+X_train_sample = X_train.sample(50, random_state=0)
+
+features = ['MedInc', 'AveOccup', 'HouseAge', 'AveRooms']
+
+print('Computing ICE plots...')
+plot_partial_dependence(est, X_train_sample, features, n_jobs=3,
+                        grid_resolution=20, individual=True)
+fig = plt.gcf()
+fig.suptitle('ICE of house value on non-location features\n'
+             'for the California housing dataset, with Gradient Boosting')
+fig.subplots_adjust(hspace=0.3)
+
+plt.show()
+
+##############################################################################
+# In ICE plots it might not be easy to see the average effect of the 'target'
+# variable. Hence, it is recommended to use ICE plots together with partial
```

Comment:
```suggestion
# variable. Hence, it is recommended to use ICE plots together with PDP
```

Reply:
Fixed in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/1dae092a99289ef463298da384ed9b64808ea933